### PR TITLE
Extra: Economics

### PR DESCRIPTION
**Pull request by smostovoy**
Economic indicators based on Quandl API
IMF database:
https://www.quandl.com/data/ODA/documentation/documentation

Syntax:
```ruby
ar = Entity('Argentina')
ar.economy.indicators
```
Indicator methods: `current`, `history`, `perediction`, and raw `data`. Properties: `unit`, `scale`, `name`

### CODE REVIEW

File: lib/reality/extras/economic.rb [hunk 1]
```diff
@@ -0,0 +1,103 @@
+require 'quandl'
+Quandl::ApiConfig.api_key = 'JU6KEPEkqqqfVkcpa9MU'
+Quandl::ApiConfig.api_version = '2015-04-09'
```
Comment (zverok) [inline lib/reality/extras/economic.rb]: I feel like we should stop this practice immediately :(
There should be some configuration block, and some `config/demo.yml`, and `Reality.configure(path)`

Comment (smostovoy) [inline lib/reality/extras/economic.rb]: sure, this is a quick 1-file draft

File: lib/reality.rb [hunk 2]
```diff
@@ -28,7 +28,8 @@ def self.reload!
   end
 
   # extras
-  require_ %w[extras/open_weather_map extras/geonames]
+  require_ %w[extras/open_weather_map extras/geonames extras/economics]
```
Comment (zverok) [inline lib/reality.rb]: Hey. Didn't you note I've changed extra modules naming from "extras/{topic}" to "extras/{service}". You should!

File: lib/reality/extras/economics/indicators.yml [hunk 3]
```diff
@@ -0,0 +1,149 @@
+---
```
Comment (zverok) [inline lib/reality/extras/economics/indicators.yml]: What/where is the source for indicators list? Can it be updated?..

File: lib/reality/extras/economics/indicator.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          return 'no data' if data.nil?
+          data.values.select { |v| v['date'] > Date.today }.sort_by { |d| d['date'] }
+        end
+
+        def data
+          @data ||= fetch
+        end
+
+        def inspect
+          "#<%s (%s)>" % ['Economics::Indicator', name]
```
Comment (zverok) [inline lib/reality/extras/economics/indicator.rb]: In fact, it is not very good to "imitate" classnames which are not real (this form of inspect implies `Economic::Indicator` is a complete classname)

**Pull request merged**